### DISCUSSION TITLE
Running Whisper on an AMD GPU

### DISCUSSION THREAD

**Discussion by RickArcher108** [openai/whisper#1258] [Q&A]
I was running the desktop version of Whisper using the CMD prompt interface successfully for a few days using the 4GB NVIDIA graphics card that came with my Dell, so I sprang for an [AMD Radeon RX 6700 XT](https://amzn.to/41jxWeD) and had it installed yesterday, only to discover that Whisper didn't recognize it and was reverting my my CPU. How can I get it to recognize the AMD GPU? I see some references to that online, but I'm 73 years old and not the most technical guy you'll ever meet, so could use a bit of help. Thanks.

**Comment by NielsMayer** [comment]
See https://github.com/openai/whisper/discussions/55#discussioncomment-4504980

**Comment by phineas-pta** [comment]
if you're using windows then no, amd only possible for linux for now

**Reply by jsmollin** [reply]
Are there instructions for getting this set up on Linux?

**Reply by phineas-pta** [reply]
not official but you can follow #55

**Comment by phineas-pta** [comment]
people managed to get it works on windows + amd: https://github.com/Const-me/Whisper

**Reply by RickArcher108** [reply]
Yes, using this desktop version I've been transcribing dozens of 2-hour interviews using the large model. Each one takes about 15-20 minutes to transcribe, and I'm very pleased with the accuracy. I wish there were a way to do batch processing, and it would be cool if it could be trained to recognize the voices and name the speakers, as Otter does (sloppily). But this is so much more accurate than Otter that it's far preferable even without those features.

**Reply by phineas-pta** [reply]
for batch processing you can try https://github.com/tigros/Whisperer

for speakers recognizing it's another difficulty level, search for "diarization" in discussions

**Comment by sergmaxin** [comment]
Great job! Everything is super. How about WhisperX?
